src/anomalib/models/components/base/anomaly_module.py

```diff
@@ -87,21 +64,6 @@ def predict_step(self, batch: Any, batch_idx: int, dataloader_idx: int = 0) -> A
         del batch_idx, dataloader_idx  # These variables are not used.
 
         outputs: Tensor | dict[str, Any] = self.validation_step(batch)
```

Comment:
Yeah, that is better. My only concern is that it will end up touching a lot more files. If it does not create extra overhead for you guys then I can make this change.

Reply:
I feel it would be an improvement. Of course, we could also do it in a separate PR. @samet-akcay curious about your opinion on this.